packages/nm-launcher/default.nix

```diff
@@ -13,7 +13,7 @@
       export DBUS_SESSION_BUS_ADDRESS=unix:path=/tmp/ssh_session_dbus.sock
       export DBUS_SYSTEM_BUS_ADDRESS=unix:path=/tmp/ssh_system_dbus.sock
       ${pkgs.openssh}/bin/ssh -M -S /tmp/control_socket \
-          -f -N -q ghaf@192.168.100.1 \
+          -f -N -q network@192.168.100.1 \
```

Comment:
These inter-VM integrations over SSH may not be long-lived but now that we are changing them, I think we better not hardcode the new system user account "network"  but use the account `e.g. users.users.network.account.user` default value (not yet declared) instead. That allows the username configuration without changing the hardcoded values. These usages, like line above, must use the nix variable instead.

Reply:
The same goes to other hard-coded usernames - `waypipe`

Reply:
Fixed